src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -312,24 +327,43 @@ def _forward(self, model_inputs):
             out = {"tokens": tokens}
         else:
             stride = model_inputs.pop("stride", None)
-            input_values = model_inputs.pop("input_values")
+            inputs = (
+                model_inputs.pop("input_values")
+                if "input_values" in model_inputs
+                else model_inputs.pop("input_features")
+            )
             attention_mask = model_inputs.pop("attention_mask", None)
-            outputs = self.model(input_values=input_values, attention_mask=attention_mask)
+            outputs = self.model(inputs, attention_mask=attention_mask)
```

Comment:
I looked more deeply into the code, the difference already exists.

`input_values` => raw input audio
`input_features` => preprocessed audio like mel spectrogram.

We already have some ifs, which we could factor out if needed.

And the biggest invariant is

`self.model(**self.feature_extractor(...))` which is still respected. The problem lies with `attention_mask` which is sometimes required but is not returned instead of returning `None`.